reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -37,10 +37,9 @@
       login: true
       challenge: false
       mapping_method: claim
-      clientID: e76865557b0417387b35 
-      clientSecret: 72b36e28221c1b93089ecf72f1a19963a8532b06 
-      organizations:
-      - openshift
+      clientID: "{{ github_client_id }}"
+      clientSecret: "{{ github_client_secret }}"
+      organizations: ['OpenShift']
```

Comment:
If you update the option to use `multiple=True`, then you should be able to just do this:
```
organizations: "{{ github_organizations }}"
```